### ISSUE TITLE
hotstar do not work now

### ISSUE THREAD

**Issue by chzuping** [rg3/youtube-dl#14647]
D:\work\youtube-dl\youtube_dl>python __main__.py -F http://www.hotstar.com/movies/ami-je-ke-tomar/1770016133
[HotStar] 1770016133: Downloading JSON metadata
[HotStar] 1770016133: Downloading JIO JSON metadata
[HotStar] 1770016133: Downloading m3u8 information
WARNING: Failed to download m3u8 information: HTTP Error 403: Forbidden

**Comment by remitamine** [comment]
most of the videos from hotstar are geo restricted.
```
python __main__.py -F http://www.hotstar.com/movies/ami-je-ke-tomar/1770016133
[HotStar] 1770016133: Downloading JSON metadata
[HotStar] 1770016133: Downloading JIO JSON metadata
[HotStar] 1770016133: Downloading m3u8 information
[info] Available formats for 1770016133:
format code  extension  resolution note
hls-304      mp4        320x180     304k , avc1.66.30, mp4a.40.2
hls-524      mp4        416x234     524k , avc1.66.30, mp4a.40.2
hls-924      mp4        640x360     924k , avc1.66.30, mp4a.40.2
hls-1424     mp4        720x404    1424k , avc1.66.30, mp4a.40.2
hls-2124     mp4        1280x720   2124k , avc1.77.30, mp4a.40.2
hls-3124     mp4        1600x900   3124k , avc1.640029, mp4a.40.2
hls-4624     mp4        1920x1080  4624k , avc1.640029, mp4a.40.2
hls-6124     mp4        1920x1080  6124k , avc1.640029, mp4a.40.2 (best)
```

**Comment by chzuping** [comment]
Thanks  very much.
